aced_submission/meta_graph_load.py

```diff
@@ -87,10 +88,23 @@ def load_vertices(files, connection, dependency_order, project_id, mapping):
             continue
         logger.info(f"loading {path} into {data_table_name}")
 
+        columns = ['node_id', '_props', 'acl', '_sysan', 'created']
```

Comment:
More of a general question — where do these column names come from, are they Gen3-specific?

Reply:
Those are the names of the `node_*` tables in gen3.

```sheepdog_local=> SELECT 
sheepdog_local->    table_name, 
sheepdog_local->    column_name, 
sheepdog_local->    data_type 
sheepdog_local-> FROM 
sheepdog_local->    information_schema.columns
sheepdog_local-> WHERE 
sheepdog_local->    table_name = 'node_patient' ;
  table_name  | column_name |        data_type         
--------------+-------------+--------------------------
 node_patient | created     | timestamp with time zone
 node_patient | _sysan      | jsonb
 node_patient | _props      | jsonb
 node_patient | acl         | ARRAY
 node_patient | node_id     | text
(5 rows)
```